### DISCUSSION TITLE
LDM Notes for July 17th, 2024

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#8307] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2024/LDM-2024-07-17.md

### Agenda

- Overload resolution priority open questions
- Better conversion from collection expression with `ReadOnlySpan<T>` overloads

**Comment by colejohnson66** [comment]
Should "`ReadOnlySpan<T>` vs `ReadOnlySpan<T>`" be "`ReadOnlySpan<T1>` vs `ReadOnlySpan<T2>`" or "`ReadOnlySpan<T>` vs `T[]`"?

**Reply by 333fred** [reply]
Thanks, fixed.
